ui/src/components/ActuatorItem.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+const props = defineProps({
+  name: {
+    type: String,
+    required: true,
+  },
+  data: {
+    type: Object,
+    required: true,
+  },
+});
+
+/**
+ * Convert given bytes to 2 digits precision round exponent version string.
+ * @param bytes number
+ */
+function convertBytes(bytes: number): string {
+  const units = ["bytes", "KB", "MB", "GB", "TB", "EB"];
+  let unitIndex = 0;
+
+  while (bytes >= 1024 && unitIndex < units.length - 1) {
+    bytes /= 1024;
+    unitIndex++;
+  }
+
+  return `${bytes.toFixed(2)} ${units[unitIndex]}`;
+}
+
+/*
+{
+   "name": "application.ready.time",
+   "description": "Time taken for the application to be ready to service requests",
+   "baseUnit": "seconds",
+   "measurements": [
+      {
+         "statistic": "VALUE",
+         "value": 2.649
+      }
+   ],
+   "availableTags": [
+      {
+         "tag": "main.application.class",
+         "values": [
+            "org.molgenis.armadillo.ArmadilloServiceApplication"
+         ]
+      }
+   ]
+}
+*/
+</script>
+<template>
+  <tr v-if="data._display" v-for="(v, k) in data.measurements">
+    <td scope="col">{{ k }}</td>
+    <td :title="data.description">
+      <span>
+        {{ data.name }}
+        <i v-if="data.description" class="bi bi-info-circle-fill"></i>
+      </span>
+    </td>
+    <td>{{ v.statistic }}</td>
+    <td v-if="data.baseUnit === 'bytes'">
+      {{ convertBytes(v.value) }}
+    </td>
+    <td v-else>{{ v.value }} {{ data.baseUnit }}</td>
+  </tr>
+  <tr v-if="data._display">
```

Comment:
Ah! This is why you use that display thingy, right? I would set this up a bit differently, like we did throughout the rest of the application. I would compute the filtered items or create a method, then only show those and in case there is no searchword, then filtered items should return the full list. If you do this, you don't have to worry about resetting things when the search is emptied. 
Example: https://github.com/molgenis/molgenis-service-armadillo/blob/23272d29bb66cc343e058587ad9dc5945897c99b/ui/src/views/Projects.vue#L310
I also changed it on the branch I created for this, so you could also look at that.

Reply:
We can check later in merged code